formats/mainstream_browse_page/frontend/examples/curated_level_2_page.json

```diff
@@ -0,0 +1,29 @@
+{
+    "base_path": "/browse/benefits/entitlement-with-list",
+    "description": "When and how benefit payments are made, benefits adviser and benefit fraud",
+    "details": {},
+    "format": "mainstream_browse_page",
+    "details": {
+      "groups": [
+        {
+          "name": "Oil rigs",
```

Comment:
It would be better to pick some examples of content which actually makes sense for "benefits entitlement", so that people don't get distracted when reviewing this by the odd content.  Also, so that merging the results from this API with the results from the content-api works.